server/routes.go

```diff
@@ -122,7 +122,7 @@ func (b *BaseRouter) addContentRoutes() {
 	}))
 
 	// Get tv details (for tv page)
-	content.Use(WhereaboutsRequired()).GET("/tv/:id", cache.CachePage(b.ms, exp, func(c *gin.Context) {
+	content.GET("/tv/:id", WhereaboutsRequired(), cache.CachePage(b.ms, exp, func(c *gin.Context) {
```

Comment:
Moved the usage of `WheraboutsRequired`. When adding it with `.Use`, the middleware will apply to all subsequently defined routes. 

Adding it as a handler before our main handler func means it will only apply to this route.

Reply:
I just realised, this might have been me that added it with `.Use` 💀